### PR TITLE
Use node lts/dubnium (v10)

### PR DESCRIPTION
**Pull request by taymoork2**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -93,6 +93,7 @@
     "ampersand-events": "^2.0.2",
     "babel-polyfill": "^6.26.0",
     "bowser": "^1.9.4",
+    "builtin-modules": "^3.1.0",
```
Comment (adamweeks) [inline package.json]: What is this for?

**Comment by adamweeks** [comment]
Could you investigate the linting and test failures?

File: package.json (hunk 1 above)
Comment (taymoork2) [reply package.json]: for some reason, [this](https://github.com/webex/react-widgets/blob/master/scripts/utils/deps.js#L6) started failing whenever I tried to build

Comment (taymoork2) [reply package.json]: this was an issue with wiping `node_modules` and `package-lock` and "starting fresh" with a `npm i`
Reverting back to the `package-lock` in the master branch and running `npm ci` corrected the issue(s)

Comment (adamweeks) [reply package.json]: Gotcha, if it is required, let's separate it into another commit with the message `fix(package): add builtin-modules package`.

**Comment by taymoork2** [comment]
eslint issues (before the revert) were pop up during jest runs
The updated `package-lock` brought these issues forward when eslint was run
```
Warning: Failed prop type: The prop `actor` is marked as required in `FileShareDisplay`, but its value is `undefined`.
        in FileShareDisplay
```

**Pull request merged**